arcgis-ios-sdk-samples/Content Display Logic/Views/AppInfoView.swift

```diff
@@ -0,0 +1,171 @@
+// Copyright 2021 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct AppInfoView: View {
+    private let links = [
+        Link(title: "Esri Community", url: URLs.esriCommunity, subtitle: "Browse and discuss in the Esri Community"),
+        Link(title: "GitHub Repository", url: URLs.githubRepository, subtitle: "Log an issue in the GitHub Repository"),
+        Link(title: "Developer Site", url: URLs.developers)
```

Comment:
This row has the same URL as poweredBys.last, so I wonder if we want to keep 2 of them.